src/TNBase.DataStorage/Service/PreferencesService.cs

```diff
@@ -0,0 +1,42 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Configuration;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TNBase.Objects;
+
+namespace TNBase.DataStorage.Service
+{
+    public class PreferencesService
+    {
+        private readonly ITNBaseContext context;
+        private readonly ApplicationSettingsBase applicationSettings;
+
+        public PreferencesService(ITNBaseContext context, ApplicationSettingsBase applicationSettings)
```

Comment:
Generally, it's not a good practice to inject the entire configuration object as that hides what dependencies have. A better option is to pass a strongly typed object with only required values, in this case, AssociationName and Logo. So in this case you could have
```suggestion
        public PreferencesService(ITNBaseContext context, PreferencesServiceOptions options)
        {
             ...
             result.ApplicationTitle = options.DefaultAssociationName;
             result.LogoFileName = options.DefaultLogo;
        }
```

By the way, I really like that you've implemented a fallback when that value is not set in the database.